### PR TITLE
Add Snowflake CLI Quickstart

### PR DESCRIPTION
**Pull request by sfc-gh-ghernandez**

### CODE REVIEW

File: site/sfguides/src/getting_started_with_snowflake_cli/getting_started_with_snowflake_cli.md [hunk 1]
```diff
@@ -0,0 +1,745 @@
...
+snow snowpark execute procedure "SNOWPARK_EXAMPLE.PUBLIC.HELLO_PROCEDURE('jdoe')"
+```
+
+Running this command should return an output similar to this one:
+```console
++-------------------------------+
+| key             | value       |
+|-----------------+-------------|
+| HELLO_PROCEDURE | Hello jdoe! |
+
```
Comment (sfc-gh-turbaszek) [inline site/sfguides/src/getting_started_with_snowflake_cli/getting_started_with_snowflake_cli.md]:
```suggestion
+-------------------------------+
```

File: site/sfguides/src/getting_started_with_snowflake_cli/getting_started_with_snowflake_cli.md [hunk 2]
```diff
@@ -0,0 +1,777 @@
...
+- You'll need a Snowflake account. You can sign up for a free 30-day trial account here: [https://signup.snowflake.com/](https://signup.snowflake.com/).
+
+### What You’ll Learn
+- How to install Snowflake CLI
+- How to configure Snowflake CLI
+- How to switch between different connections
+- How to download and upload files using Snowflake CLI
+- How to execute SQL using Snowflake CLI
+
+Ensure your development environment meets the following requirements before proceeding:)
```
Comment (sfc-gh-ghernandez) [inline site/sfguides/src/getting_started_with_snowflake_cli/getting_started_with_snowflake_cli.md]: @sfc-gh-turbaszek Do we need this section?

File: site/sfguides/src/getting_started_with_snowflake_cli/getting_started_with_snowflake_cli.md [hunk 3]
```diff
@@ -0,0 +1,777 @@
...
+│ --role,--rolename                   TEXT  Role to use. Overrides the value specified for the connection.                                                                                                │
+│ --warehouse                         TEXT  Warehouse to use. Overrides the value specified for the connection.                                                                                           │
+│ --temporary-connection      -x            Uses connection defined with command line parameters, instead of one defined in config                                                                        │
+│ --mfa-passcode                      TEXT  Token to use for multi-factor authentication (MFA)                                                                                                            │
+╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+```
+
+You can access this list by running any Snowflake CLI command with `--help`.
+
```
Comment (sfc-gh-ghernandez) [inline site/sfguides/src/getting_started_with_snowflake_cli/getting_started_with_snowflake_cli.md:225]: @sfc-gh-turbaszek I didn't quite understand this section. Let's chat offline.

File: site/sfguides/src/getting_started_with_snowflake_cli/getting_started_with_snowflake_cli.md (hunk 2 above)
Comment (sfc-gh-turbaszek) [reply site/sfguides/src/getting_started_with_snowflake_cli/getting_started_with_snowflake_cli.md]: Good catch, we don't need it

**Pull request merged**